site/content/docs/5.1/components/popovers.md

```diff
@@ -48,15 +48,15 @@ var popoverList = popoverTriggerList.map(function (popoverTriggerEl) {
 
 We use JavaScript similar to the snippet above to render the following live popover. Titles are set via `title` attribute and body content is set via `data-bs-content`.
 
-{{< example >}}
+{{< example javascript="popover-demo.js" >}}
 <button type="button" class="btn btn-lg btn-danger" data-bs-toggle="popover" title="Popover title" data-bs-content="And here's some amazing content. It's very engaging. Right?">Click to toggle popover</button>
```

Comment:
Don't know why but the code contains `title="Popover title"`, it works in the doc but I can't find the `title` in the DOM and so it is missing in StackBlitz as well :thinking:  If I replace `title="Popover title"` by `data-bs-title="Popover title"`, everything's working correctly but I don't know the impact of such a change.

Reply:
Your answer is [here](https://github.com/twbs/bootstrap/blob/51535cd95ac8eb5746e19c057aeabdbcafef3a8b/js/src/tooltip.js#L513) unfortunately :disappointed:

Reply:
OK so all popovers that contains titles won't be exportable in StackBlitz. The button should be hidden for those cases. Except if someone thinks about a way of doing it differently.

Reply:
This one is describing the `title` attribute fallback process, so it may is better to keep it as it was (no btn)

If we have other popovers using `title`, I think it's ok to change them using `data-bs-title`